res/controllers/common-hid-packet-parser.js

```diff
@@ -1075,6 +1087,15 @@ HIDController.prototype.processIncomingPacket = function(packet,delta) {
     }
 }
 
+HIDController.prototype.isParameterControl = function(control) {
```

Comment:
the "myscaler.useSetParameter = true;" thing works fine -- I use the same scaler for all parameter knobs so I only have to set it once.